src/data/dynamic/form/Leads.js

```diff
@@ -0,0 +1,199 @@
+import { AGES, MAJORS } from "./Information";
+import { GRADES, GENDERS, SHIRTS, SOURCES } from "./Information";
+import { SCHOOLS } from "./Schools";
+
+export const FIELDS = {
+  description: {
+    input: "description",
+    width: 12,
+    texts: [
+      "Welcome to HACKATHON_NAME. Thank you for considering to become a lead, we appreciate your enthusiasm to join HACKATHON_NAME. HACKATHON_NAME is a DESCRIPTION hackathon spanning HACKATHON_LENGTH on HACKATHON_DATE.",
+    ],
+  },
+  name: {
+    input: "input",
+    name: "name",
+    type: "text",
+    title: "Name",
+    maxLength: 50,
+    width: 12,
+    editable: false,
+    required: true,
+  },
+  preferredName: {
+    input: "input",
+    name: "preferredName",
+    type: "name",
+    title: "Preferred Name",
+    maxLength: 50,
+    width: 12,
+    editable: true,
+    required: true,
+  },
+  email: {
+    input: "input",
+    name: "email",
+    type: "email",
+    title: "Email Address",
+    maxLength: 50,
+    width: 12,
+    editable: false,
+    required: true,
+  },
+  phone: {
+    input: "input",
+    name: "phone",
+    type: "phone",
+    title: "Phone Number",
+    placeholder: "123 456 7890",
+    maxLength: 50,
+    width: 12,
+    required: true,
+  },
+  discord: {
+    input: "input",
+    name: "discord",
+    type: "text",
+    title: "Discord Username",
+    placeholder: "john_doe#1234",
+    maxLength: 50,
+    width: 12,
+    required: true,
+  },
+  major: {
+    input: "select",
+    title: "Major",
+    options: MAJORS,
+    field: "major",
+    placeholder: "ie. Computer Science",
+    width: 12,
+    required: true,
+    searchable: true,
+  },
+  age: {
+    input: "select",
+    title: "Age",
+    options: AGES,
+    field: "age",
+    placeholder: "Age",
+    width: 12,
+    required: true,
+    searchable: true,
+  },
+  school: {
+    input: "select",
+    title: "School",
+    options: SCHOOLS,
+    field: "school",
+    placeholder: "School",
+    width: 12,
+    required: true,
+    searchable: true,
+  },
+  grade: {
+    input: "select",
+    title: "Grade",
+    options: GRADES,
+    field: "grade",
+    placeholder: "ie. Undergraduate",
+    width: 12,
+    required: true,
+    searchable: true,
+  },
+  gender: {
+    input: "radio",
+    text: "Gender",
+    options: GENDERS,
+    field: "gender",
+    width: 12,
+    required: true,
+  },
+  shirt: {
+    input: "radio",
+    text: "Shirt Size",
+    options: SHIRTS,
+    field: "shirt",
+    width: 12,
+    required: true,
+  },
+  eventSource: {
+    input: "select",
+    title: "How did you find the event?",
+    options: SOURCES,
+    field: "eventSource",
+    placeholder: "Event Source",
+    width: 12,
+    required: true,
+  },
+  priorHackathons: {
+    input: "checkboxes",
+    text: "Prior Hackathons",
+    field: "priorHackathons",
+    width: 12,
+    required: true,
+    options: [
+      "Cutie Hack",
+      "Rose Hack",
+      "Designverse",
```

Comment:
DesignVerse

Reply:
maybe add MLH as an option and then other hackathons in case they didn't find out thru UCR hackathons